ykrt/build.rs

```diff
@@ -20,17 +21,18 @@ fn main() {
         let mut rustflags = env::var("RUSTFLAGS").unwrap_or_else(|_| String::new());
         let tracing_kind = find_tracing_kind(&rustflags);
         rustflags = make_internal_rustflags(&rustflags);
-        let status = Command::new(cargo)
-            .current_dir(internal_dir.join("ykshim"))
+        let mut cmd = Command::new(cargo);
```

Comment:
Can this be `let mut cmd = Command::new(cargo).current_dir(...)...`?

Reply:
Afraid not:
```
24 |         let mut cmd = Command::new(cargo)
   |                       ^^^^^^^^^^^^^^^^^^^ creates a temporary which is freed while still in use
...
29 |             .env("RUSTFLAGS", &rustflags);
   |                                          - temporary value is freed at the end of this statement
...
32 |             cmd.arg("--release");                                                                 
   |             --- borrow later used here
   |
   = note: consider using a `let` binding to create a longer lived value
```

Reply:
Fair enough!